<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SeniorCodeSetUI</class>
 <widget class="QDialog" name="SeniorCodeSetUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>177</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ButelLive</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(35, 35, 37);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="TitleBar" name="m_pTitleLbl" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(201, 201, 201);</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_40">
          <property name="text">
           <string>H264CPU Preset：</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="m_pCombCPUPreset">
          <property name="styleSheet">
           <string notr="true">QComboBox::drop-down{
	border-image: url(:images/arrow_down.png);
    width:10px;
    height:10px;
    subcontrol-origin: padding;
    subcontrol-position: center right;
    margin: 14 8px;
}
QComboBox{
    height:20px;
    border: 1px solid #dddddd;
    background-color: rgb(221,221,221);
    color: #2a2a2a;
	border-radius: 4px;
}</string>
          </property>
          <item>
           <property name="text">
            <string>ultrafast</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>superfast</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>veryfast</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>faster</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>fast</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>medium</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>slow</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>slower</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_41">
          <property name="text">
           <string>Code configuration file：</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="m_pCombCodeCfgFile">
          <property name="styleSheet">
           <string notr="true">QComboBox::drop-down{
	border-image: url(:images/arrow_down.png);
    width:10px;
    height:10px;
    subcontrol-origin: padding;
    subcontrol-position: center right;
    margin: 14 8px;
}
QComboBox{
    height:20px;
    border: 1px solid #dddddd;
    background-color: rgb(221,221,221);
    color: #2a2a2a;
	border-radius: 4px;
}</string>
          </property>
          <item>
           <property name="text">
            <string>main</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>high</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_42">
          <property name="text">
           <string>Keyframe interval (SEC, 0= automatically)：</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="m_pSBoxKeyFrameInterval"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_43">
          <property name="text">
           <string>The number of B frames between key frames (3= default)：</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QSpinBox" name="m_pSBoxBFrameNum"/>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="m_pBtnOk">
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>20</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);
border-image: url(:/images/btn_back.png);</string>
          </property>
          <property name="text">
           <string>OK</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_pBtnCancel">
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>20</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);
border-image: url(:/images/btn_back.png);</string>
          </property>
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TitleBar</class>
   <extends>QWidget</extends>
   <header location="global">titlebar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
